src/WatchDogs.Test/DxTradeAuthenticatorTests.cs

```diff
@@ -0,0 +1,81 @@
+﻿using Contracts;
+using Infrastructure.DxTrade;
+using Microsoft.Extensions.Options;
+using Moq;
+using RichardSzalay.MockHttp;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Security.Authentication;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WatchDogs.Test;
+public class DxTradeAuthenticatorTests
```

Comment:
Is is good practice to have an empty line between namespace and class declaration.

Reply:
Done.